sre-wrapper/job_wrapper.py

```diff
@@ -136,9 +136,11 @@ def run(input_id: str = None):
 
 
 @cli.command()
-def upload_result():
+@click.option("--reference-target", help="Dotted id to an analysis entity's job result. Should be on the format: entityId.jobs.*index*.result.", type=str, required=True)
```

Comment:
```suggestion
@click.option("--reference-target", help="Dotted id to an analysis entity's job result. Should be on the format: entityId.jobs.*index*.result.", type=str, required=True)
```
This help text is kind of wrong. It's just a dotted Id to where to insert a reference, which ever entity that is.
But it might be helpful to write how we are using it in analysisPortal